cirq/linalg/transformations.py

```diff
@@ -471,7 +477,7 @@ def sub_state_vector(
     if default is not RaiseValueErrorIfNotProvided:
         return default
 
-    raise ValueError(
+    raise UnfactorizableStateVectorError(
```

Comment:
Need to update Raises section of the function docstring above.

Reply:
good point